src/bench/coin_selection.cpp

```diff
@@ -29,7 +29,7 @@ static void addCoin(const CAmount& nValue, const CWallet& wallet, std::vector<st
 static void CoinSelection(benchmark::Bench& bench)
 {
     auto chain = interfaces::MakeChain();
-    const CWallet wallet(*chain, WalletLocation(), CreateDummyWalletDatabase());
+    const CWallet wallet(*chain, "", CreateDummyWalletDatabase());
```

Comment:
why can we not fully backport 19619?

Reply:
The rest of 19619 will be backported in the second series of pull requests, it can be dropped due to it being partial if needed

Reply:
that would probably be better if you can backport it in 1 piece soon